src/token-processor/blockchain-api/blockchain-smart-contract-monitor.ts

```diff
@@ -77,7 +77,9 @@ export class BlockchainSmartContractMonitor {
             try {
               await this.handleSmartContract(messageJson.payload);
             } catch (error) {
-              logger.error(`BlockchainSmartContractMonitor error handling contract deploy`, error);
+              logger.error(
```

Comment:
What's the reason for these changes? I think usually the default `Error.toString()` that this will use only includes the error `message` property (so the stack trace will be missing from these logs). In the API's winston logger I think we usually do ``log(`got error: ${error}`, error)`` which includes the error message in the main string, then also includes the error stacktrace and other properties in the log entry metadata.

Reply:
It's because the error message did not appear on logs but you're right, I'm adding it as the cause as well

Reply:
It looks like the way to do it with pino is just include the metadata object (error instance in this case) as the first param, and the message as the second param: https://github.com/pinojs/pino/blob/master/docs/api.md#mergingobject